### PR TITLE
show executeTime and passRate in autotest-plan list

### PR DESCRIPTION
**Pull request by ZLValien**
#### What type of this PR

/kind feature

#### What this PR does / why we need it:

add executeTime and passRate in autotest-plan table component

### CODE REVIEW

File: apistructs/testplan_v2.go [hunk 1]
```diff
@@ -23,19 +23,21 @@ import (
 
 // TestPlanV2 testplan
 type TestPlanV2 struct {
```
Comment (sfwn) [inline apistructs/testplan_v2.go:25]: AutoTestPlan

File: modules/openapi/component-protocol/scenarios/auto-test-plan-list/protocol.yml [hunk 2]
```diff
@@ -86,6 +89,14 @@ components:
           dataIndex: "testSpace"
         - title: "负责人"
           dataIndex: "owners"
+        - title: "最近执行通过率"
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/auto-test-plan-list/protocol.yml:92]: add TODO for i18n

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/testplan_v2.go (hunk 1 above)
Comment (ZLValien) [reply apistructs/testplan_v2.go:26]: add TODO

File: modules/openapi/component-protocol/scenarios/auto-test-plan-list/protocol.yml (hunk 2 above)
Comment (ZLValien) [reply modules/openapi/component-protocol/scenarios/auto-test-plan-list/protocol.yml:93]: add TODO

**Review by sfwn — APPROVED** [review]

**Pull request merged**